javav2/example_code/cloudfront/Readme.md

```diff
@@ -4,67 +4,173 @@ This README discusses how to run and test the Java code examples for Amazon Clou
 
```

Comment:
Overall comment on the README: Recommend following the format in [this template](https://github.com/awsdocs/aws-doc-sdk-examples/wiki/Service-level-README-template).